topics/howtosim.rst

```diff
@@ -46,83 +47,79 @@ Parameters for ``gendisk`` are specified in a YAML file with the following field
+   SCSPH                : XXXXX                # Scale for Spherical SL coordinate mapping - this should match rmapping in exp
    SEED                 : XXXXX                # Random number seed (e.g. your favorite number)
    SELECT               : false                # Enable significance selection in coefficient computation
    SHFAC                : 16                   # Scale height factor for assigning vertical table size
-   SVD                  : false                # Use svd for symmetric eigenvalue problesm
    TNUM                 : 80                   # Number of cos(theta) knots for EmpCylSL basis construction quadrature
-   Temp                 : 2000                 # Gas temperature (in K)
-   Tmin                 : 500                  # Temperature floor (in K) for gas disk generation
    ToomreQ              : XXXXX                # Toomre Q parameter for stellar disk generation (e.g. 1.4)
-   U0                   : 0                    # Disk-Halo x velocity center position
-   V0                   : 0                    # Disk-Halo y velocity center position
-   VFLAG                : 16                   # Output flags for EmpCylSL
-   W0                   : 0                    # Disk-Halo z velocity center position
-   X0                   : 0                    # Disk-Halo x center position
-   Y0                   : 0                    # Disk-Halo y center position
-   Z0                   : 0                    # Disk-Halo z center position
+   U0                   : 0.0                  # Disk-Halo x velocity center position
+   V0                   : 0.0                  # Disk-Halo y velocity center position
+   VFLAG                : 8                    # Output flags for EmpCylSL
+   W0                   : 0.0                  # Disk-Halo z velocity center position
+   X0                   : 0.0                  # Disk-Halo x center position
+   Y0                   : 0.0                  # Disk-Halo y center position
+   Z0                   : 0.0                   # Disk-Halo z center position
    basis                : false                # Print out disk and halo basis
-   ignore               : XXXXX                # 'false' if you want to generate and save out cache files, otherwise 'true'
+   ignore               : XXXXX                # 'true' if you want to generate and save out cache files, otherwise 'false'
    cachefile            : XXXXX                # Name of EOF cache file (e.g. eof.cache.fileF)
    centerfile           :                      # Read position and velocity center from this file
    constheight          : true                 # Use constant disk scale height
    dbods                : disk.bods            # Disk particle output file
-   deproject            : Exponential          # set deprojection model (Exponential or MN)
+   deproject            : EXP                  # set deprojection model (Exponential or MN)
```

Comment:
Technically, the code only checks for `MN` input right now, and then defaults to exponential. So we should set this with whatever is clearest and update documentation elsewhere.